client_packages/index.js

```diff
@@ -1 +1,2 @@
+mp.gui.execute("window.location = 'package://chat/index.html'");
```

Comment:
It's better to create ./index.js on chat folder, and then call chat in the main index like that: `require('./chat'); `